internal/wrappers/projects-http.go

```diff
@@ -148,6 +149,31 @@ func (p *ProjectsHTTPWrapper) GetByID(projectID string) (
 	return handleProjectResponseWithBody(resp, err, http.StatusOK)
 }
 
+func (p *ProjectsHTTPWrapper) GetByName(name string) (
```

Comment:
you didn't use this func on code. please remove it

Reply:
It is used in integration tests